nservicebus/pipeline/features.md

```diff
@@ -114,6 +105,29 @@ snippet:FeatureSetup
 Note: Features are automatically detected and registered by NServiceBus when the assembly is scanned.
 
 
+## Feature settings
+
+
+The settings are a good way to share data and configuration between the endpoint configuration or other features. Written settings are then available in the `FeatureConfigurationContext.Settings` property during the *setup phase*. Settings can be written via *defaults* or the `EndpointConfiguration`:
+
+### Defaults
+
+When a feature is found to be enabled, the bootstrapping code applies the *defaults* defined by that feature to a shared dictionary containing the settings.
+
+snippet:FeatureWithDefaults
+
+The code above configures the key `"Key"` to contain value `"Value"` and the key `"OtherKey"` to contain value `42` as a default value. The querying API allows to distinguish these two types of registrations. Usually, for a given key, a feature registers a default value in this way and also exposes an extension method of the endpoint configuration to allow a user to override this default value.
+
+
+### EndpointConfiguration
+
+The settings can already be accessed during endpoint configuration:
+
+snippet:WriteSettingsFromEndpointConfiguration
+
+Note that *defaults* have not yet been applied at endpoint configuration time.
```

Comment:
@timbussmann But when it comes to "defaults" and "custom/overrides", the latter always takes precedence, irrelevant of the fact in which order they were applied. Correct?